zanata-war/src/test/java/org/zanata/ZanataDbunitJpaTest.java

```diff
@@ -48,33 +48,16 @@
 
     private boolean prepared = false;
 
-    @BeforeClass
-    @Parameters("binaryDir")
-    public void setBinaryDir(@Optional String binaryDir) {
-        if (binaryDir == null)
-            return;
-        this.binaryDir = binaryDir;
-    }
-
     protected String getBinaryDir() {
         return binaryDir;
     }
 
-    @BeforeClass
-    @Parameters("replaceNull")
-    public void setReplaceNull(@Optional Boolean replaceNull) {
-        if (replaceNull == null)
-            return;
-        this.replaceNull = replaceNull;
-    }
```

Comment:
On 2015-06-02 13:58, Carlos Munoz wrote:
> In zanata-war/src/test/java/org/zanata/ZanataDbunitJpaTest.java
> <https://github.com/zanata/zanata-server/pull/823#discussion_r31491577>:
> 
>>      protected String getBinaryDir() {
>>          return binaryDir;
>>      }
>>  
>> -    @BeforeClass
>> -    @Parameters("replaceNull")
>> -    public void setReplaceNull(@Optional Boolean replaceNull) {
>> -        if (replaceNull == null)
>> -            return;
>> -        this.replaceNull = replaceNull;
>> -    }
> 
> So, these methods are no longer needed... :question:

They haven't really been needed since we stopped injecting test
parameters via xml files.  Which always made IDE testing difficult, anyway.


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/823/files#r31491577>.
> 


-- 
Sean Flanigan

Principal Software Engineer
Localisation Tools Engineering
Red Hat